### PR TITLE
add minecraft hero to 'start new project' widget

### PR DESCRIPTION
**Pull request by davidsbailey**
#### before
screen shot 2017-11-14 at 10 27 49 am 1

#### after
screen shot 2017-11-14 at 4 01 30 pm

### CODE REVIEW

**Comment by caleybrock** [comment]
This image has been approved then? 

Will this project type be allowed to be selected in the script edit page on levelbuilder?

**Comment by caleybrock** [comment]
This is probably a @poorvasingal or mark question, but the fact that the margins above and below the category names makes this list really hard to parse visually. I get that it takes up a lot of space, but I think it would be better to double the margin above the categories.

**Comment by davidsbailey** [comment]
Thanks Caley, great questions.

yes, @ryansloan has gotten approval for this image.

No, I have overlooked levelbuilder yet again. Thanks for catching this! I've fixed it now.

I'll wait for PM/UX feedback before merging.

File: apps/src/lib/script-editor/ScriptEditor.jsx [hunk 1]
```diff
@@ -264,6 +264,7 @@ const ScriptEditor = React.createClass({
             <option value="frozen">Frozen</option>
             <option value="minecraft_adventurer">Minecraft Adventurer</option>
             <option value="minecraft_designer">Minecraft Designer</option>
+            <option value="minecraft_hero">Minecraft Hero</option>
```
Comment (caleybrock) [inline apps/src/lib/script-editor/ScriptEditor.jsx:267]: 👍

**Comment by caleybrock** [comment]
I think it's fine to merge this and do the style fixing in a follow up PR - it's mostly unrelated to this.

**Comment by davidsbailey** [comment]
OK, will do. @poorvasingal , we'd still love your input.

**Comment by davidsbailey** [comment]
Here it is with double the margin between sections:
screen shot 2017-11-14 at 4 29 50 pm
